src/core/qgspathresolver.cpp

```diff
@@ -163,9 +164,20 @@ QString QgsPathResolver::writePath( const QString &src ) const
     return src;
   }
 
-  // Strip "file://"
-  QFileInfo srcFileInfo( src.startsWith( QStringLiteral( "file://" ) ) ? src.mid( 7 ) : src );
-  QString srcPath = srcFileInfo.exists() ? srcFileInfo.canonicalFilePath() : src;
+  // Check if it is a plublicSource uri and clean it
```

Comment:
Public

Reply:
Gosh, I fixed a typo and I made another one :)